.scrutinizer.yml

```diff
@@ -0,0 +1,19 @@
+filter:
+    excluded_paths: [vendor/*, Tests/*, bin/*, Resources/]
+checks:
+  php: true
+tools:
+  external_code_coverage:
```

Comment:
Wrong identation here...
All other options are processed like subnodes of tools.external_code_coverage

```
Config Error: Unrecognized options "php_sim, php_mess_detector, php_cs_fixer, php_analyzer, php_code_sniffer, sensiolabs_security_checker, php_loc, php_pdepend" under "{root}.tools.external_code_coverage"
```

https://scrutinizer-ci.com/g/goaop/goaop-symfony-bundle/inspections/8192f7bb-62a8-4049-86e9-941ccfa1bd9e/log